test/jdk/com/sun/nio/sctp/SctpChannel/CloseDescriptors.java

```diff
@@ -26,6 +26,8 @@
  * @bug 8238274
  * @summary Potential leak file descriptor for SCTP
  * @requires (os.family == "linux")
+ * @library /test/lib
+ * @build jtreg.SkippedException
```

Comment:
I've totally missed that `jtreg.SkippedException` should indeed be added to `@build`. Now I can land a PR touching thousands of test files using `SE`. :smiling_imp: (No, I won't. :sweat_smile:)